### DISCUSSION TITLE
Help, that's all i need.    (using Steam Deck)

### DISCUSSION THREAD

**Discussion by KingGoblin2594** [community/community#131690] [New to GitHub]
Running inside Flatpak sandbox, version 1.15.6.
Found Steam directory at /home/deck/.local/share/Steam
Using default Steam Runtime at /home/deck/.local/share/Steam/ubuntu12_32/steam-runtime
WINETRICKS environment variable is not available. Searching from $PATH.
Found 2 Steam library folders
Currently logged-in Steam user: that_guy40
Found 2 Steam shortcuts running using Steam compatibility tools
Using 'yad' as GUI provider

What mean, how fix?

**Comment by ghostinhershell** [comment]
Hi @KingGoblin2594,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [Steam Deck - Steam Support page](https://help.steampowered.com/en/wizard/HelpWithSteamDeck). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
